langstream-api-gateway-auth/langstream-google-api-gateway-auth/src/main/java/ai/langstream/apigateway/auth/impl/google/GoogleAuthenticationProvider.java

```diff
@@ -59,7 +59,11 @@ public void initialize(Map<String, Object> configuration) {
     @Override
     public GatewayAuthenticationResult authenticate(GatewayRequestContext context) {
         try {
-            GoogleIdToken idToken = verifier.verify(context.credentials());
+            final String credentials = context.credentials();
+            if (credentials == null) {
+                return GatewayAuthenticationResult.authenticationFailed("Token not found.");
```

Comment:
Maybe it is better to say "Credentials not found", token is specific to some mechanisms